assets/js/components/AnalyticsInactiveCTA.js

```diff
@@ -24,58 +24,24 @@ import PropTypes from 'prop-types';
 /**
  * WordPress dependencies
  */
-import { useCallback } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
-import {
-	showErrorNotification,
-} from '../util';
-import Data from 'googlesitekit-data';
-import CTA from './legacy-notifications/cta';
-import GenericError from './legacy-notifications/generic-error';
-import { STORE_NAME as CORE_USER, PERMISSION_MANAGE_OPTIONS } from '../googlesitekit/datastore/user/constants';
-import { STORE_NAME as CORE_MODULES } from '../googlesitekit/modules/datastore/constants';
-
-const { useSelect, useDispatch } = Data;
-
+import ActivateModuleCTA from './ActivateModuleCTA';
 function AnalyticsInactiveCTA( props ) {
 	const {
 		title = __( 'Learn more about what visitors do on your site.', 'google-site-kit' ),
 		description = __( 'Connect with Google Analytics to see unique visitors, goal completions, top pages and more.', 'google-site-kit' ),
 		ctaLabel = __( 'Set up Analytics', 'google-site-kit' ),
```

Comment:
This should be removed as well, since it's not actually relevant. Please also check any usages of `AnalyticsInactiveCTA` to remove the prop if passed.

Reply:
Updated @felixarntz